### ISSUE TITLE
Harmonize debug names and provide helper in admin log window

### ISSUE THREAD

**Issue by tkurki** [SignalK/signalk-server#1031]
Over time different software components have adopted `debug` module in the server, plugins and related components.

Now we have the debug log available in the server UI, but activating specific debug logs is opaque because you need either to 
- activate all, see the keys and then pick the one you need or
- dig into js code to find the key

We should harmonise the debug names and then provide a list of relevant names in the log display ui to help the users and developers to find the relevant names.

**Comment by tkurki** [comment]
```
require('debug').instances.forEach(i => console.log(i.namespace))

@essense/instrument-config
@signalk/set-system-time
@signalk/signalk-to-nmea0183
@signalk/zones
canboatjs:canbus
canboatjs:candevice
canboatjs:discovery
canboatjs:fromPgn
canboatjs:fromPgn:trace
canboatjs:ikonvert
canboatjs:venus
canboatjs:ydgw02
n2k-signalk-126983
n2k-signalk-126985
n2k-signalk-129284
n2k-signalk-129285
serialport/binding-abstract
serialport/binding-abstract
serialport/bindings
serialport/bindings
serialport/bindings/poller
serialport/bindings/poller
serialport/bindings/unixRead
serialport/bindings/unixRead
serialport/bindings/unixWrite
serialport/bindings/unixWrite
serialport/stream
signalk-derived-data
signalk-parser-nmea0183/APB
signalk-parser-nmea0183/APB
signalk-parser-nmea0183/APB
signalk-parser-nmea0183/APB
signalk-parser-nmea0183/APB
signalk-parser-nmea0183/APB
signalk-parser-nmea0183/DBT
signalk-parser-nmea0183/DBT
signalk-parser-nmea0183/DBT
signalk-parser-nmea0183/DBT
signalk-parser-nmea0183/DBT
signalk-parser-nmea0183/DBT
signalk-parser-nmea0183/DSC
signalk-parser-nmea0183/GGA
signalk-parser-nmea0183/GLL
signalk-parser-nmea0183/HDG
signalk-parser-nmea0183/HDG
signalk-parser-nmea0183/MWV
signalk-parser-nmea0183/PBVE
signalk-parser-nmea0183/PNKEP
signalk-parser-nmea0183/VDM
signalk-parser-nmea0183/ZDA
signalk-parser-nmea0183/getTagBlock
signalk-provider-tcp
signalk-provider-tcp.data
signalk-server
signalk-server
signalk-server-node/providers/nmea0183-signalk
signalk-server:config
signalk-server:deltacache
signalk-server:discovery
signalk-server:interfaces:applicationData
signalk-server:interfaces:mdns
signalk-server:interfaces:rest
signalk-server:interfaces:tcp
signalk-server:interfaces:tcpstream
signalk-server:interfaces:ws
signalk-server:put
signalk-server:requestResponse
signalk-server:serverroutes
signalk-server:subscriptionmanager
signalk-server:udp-provider
signalk-to-influxdb
signalk:actisense-out
signalk:canbusjs
signalk:canbusjs
signalk:executor
signalk:interfaces:appstore
signalk:interfaces:logfiles
signalk:interfaces:plugins
signalk:interfaces:webapps
signalk:logging
signalk:modules
signalk:n2k-signalk
signalk:n2kAnalyzer
signalk:provider:gpsd
signalk:serialport
signalk:simple
spdy:client
spdy:connection:client
spdy:connection:server
spdy:framer
spdy:framer
spdy:framer:extra
spdy:priority
spdy:scheduler
spdy:server
spdy:stream:client
spdy:stream:server
spdy:window:client
spdy:window:server
venus-mqtt
```

**Comment by tkurki** [comment]
Fixed in #1124
